### ISSUE TITLE
Setup a simple self-service way a donor can request a duplicate receipt

### ISSUE THREAD

**Issue by ScottDowne** [MozillaFoundation/Advocacy#397]
One request we get in the donate@ inbox is "I can't find my receipt. Can you send me a new one for my tax preparation?"

User story:
"As a donor, I want to request a duplicate receipt be sent to me by email."
1) We should provide instructions on how to do this in PayPal; and
2) We should provide a "click here for duplicate receipt" simple interface or even just a URL we put in our FAQ and provide via Desk.com auto-reply.

Migrated from https://github.com/mozilla/donate.mozilla.org/issues/791

**Comment by valianttry** [comment]
Related to #411

**Comment by valianttry** [comment]
@stephaniemcv I suggest we Blue Sky this, I don't think we have capacity to build this at this stage.

**Comment by ScottDowne** [comment]
Agreed. It's also blocked on a few other things, like emails from Salesforce and the FAQ.
